custom/icds_reports/ucr/data_sources/tasks_cases.json

```diff
@@ -262,7 +262,76 @@
         "type": "boolean",
         "display_name": null,
         "column_id": "open_child_count"
-      }
+      },
+	  {
+	    "datatype": "integer", 
+		"type": "expression", 
+		"is_nullable": true, 
+		"expression": {
+		  "test": {
+			"operator": "eq", 
+			"type": "boolean_expression", 
+			"expression": {
+			  "value_expression": {
+				"datatype": null, 
+				"type": "property_name", 
+				"property_name": "migration_status"
+			  }, 
+			  "type": "related_doc", 
+			  "related_doc_type": "CommCareCase", 
+			  "doc_id_expression": {
+				"type": "icds_parent_parent_id"
+			  }
+			}, 
+			"property_value": "migrated"
+		  }, 
+		  "expression_if_true": {
+			"constant": 1, 
+			"type": "constant"
+		  }, 
+		  "type": "conditional", 
+		  "expression_if_false": {
+			"constant": 0, 
+			"type": "constant"
+		  }
+		}, 
+		"column_id": "is_migrated"
+	  },
+	  {
+	  "datatype": "integer", 
+		"type": "expression", 
+		"is_nullable": true, 
+		"expression": {
+		  "test": {
+			"operator": "not_eq", 
+			"type": "boolean_expression", 
+			"expression": {
+			  "value_expression": {
+				"datatype": null, 
+				"type": "property_name", 
+				"property_name": "registered_status"
+			  }, 
+			  "type": "related_doc", 
+			  "related_doc_type": "CommCareCase", 
+			  "doc_id_expression": {
+				"type": "named", 
+				"name": "parent_id"
```

Comment:
I think that registered_status is on the person case, correct? So this should use icds_parent_parent_id as well?

Reply:
Ah, right. Making that change now.